### PR TITLE
Install Astro on AWS Updates

### LINKED ISSUE #903: Updates to Install Astro on AWS
In a [Slack conversation](https://astronomer.slack.com/archives/C015V2JFKT5/p1657577471659509) @ben-astro made the following request:

I have a couple recommendations for [Step 3 in the AWS install Guide](https://docs.astronomer.io/astro/install-aws#step-3-create-a-cross-account-iam-role-for-astro) (see image), referencing the instructions in the AWS Management Console tab:

- The first line “After your Astro Cluster is created” is potentially misleading; technically the cluster doesn’t exist yet, and can’t be created until after the Cross Account Role is created. Maybe we can adjust this to something like the following?

“Once you provide [Activation details](https://docs.astronomer.io/astro/data-plane-activation#pre-flight-checklist) to Astronomer, Astronomer will provide you with an external ID.”

- Step 1 - Create an AWS Admin IAM user and user group - I think this can be deleted. The user should have already been created during [prerequisites](https://docs.astronomer.io/astro/install-aws#prerequisites), and I don’t know of any reason why we need a user group created. Maybe the link to AWS IAM can be provided in the prereqs section as assisting material there?

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #903

### CODE REVIEW

File: astro/install-aws.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ For a complete list of the AWS resources that our team will provision in your AW
 Before completing this setup, make sure that you have:
 
 - A dedicated AWS account with minimum EC2 service quotas.
+- An AWS administrator IAM user and a user group. See [Creating an administrator IAM user and user group (console)](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html#getting-started_create-admin-group-console)
```
[the remark this answers is not in the document]
Comment (ben-astro) [reply astro/install-aws.md:27]: Also, this bullet is covering the same need as what's listed in the next bullet ("a user with the following permissions") - I think you can delete this first bullet ("An AWS administrator.....") in total, and just link the AWS doc about creating admin IAM users (as a reference) in the bullet below.

File: astro/install-aws.md [hunk 2]
```diff
@@ -100,7 +101,7 @@ From here, our team will provision an Astro cluster according to the specificati
 
 ## Step 3: Create a cross-account IAM role for Astro
 
-After your Astro Cluster is created, Astronomer provides you with an external ID. Save the external ID as a secret or in another secure format. You need to add the external ID to AWS to allow Astronomer to access your AWS resources. For more information, see [How to use an external ID when granting access to your AWS resources to a third party](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
+To create a cross-account IAM role for Astro you need an external ID to allow Astronomer to access your AWS resources. To obtain your external ID, provide your [activation details](data-plane-activation.md#pre-flight-checklist) to [Astronomer support](https://support.astronomer.io). When you receive the external ID, save it as a secret or in another secure format. See [How to use an external ID when granting access to your AWS resources to a third party](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
```
[the remark this answers is not in the document]
Comment (ben-astro) [reply astro/install-aws.md:104]: want to make sure it's explicit that we will only touch resources in the dedicated account they provide us

[the remark this answers is not in the document]
Comment (kwcanuck) [reply astro/install-aws.md:104]: @ben-astro the text has been updated.

File: astro/install-aws.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kwcanuck) [reply astro/install-aws.md:27]: @ben-astro I removed the bullet and moved the link as discussed.

**Comment by kwcanuck** [comment]
@ben-astro the edits you requested have been added. Please review the changes and provide your edits or approval. Thanks!

**Review by ben-astro — APPROVED** [review]
LGTM!

File: astro/install-aws.md [hunk 3]
```diff
@@ -33,6 +33,8 @@ Before completing this setup, make sure that you have:
     - `PutRolePolicy`
     - `ListRoles`
     - `UpdateAssumeRolePolicy`
+    
+  See [Creating an administrator IAM user and user group (console)](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html#getting-started_create-admin-group-console)
```
Comment (jwitz) [inline astro/install-aws.md:37]:
```suggestion
   See [Creating an administrator IAM user and user group (console)](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html#getting-started_create-admin-group-console).
```

Comment (jwitz) [reply astro/install-aws.md:37]: Might be worth clarifying "An IAM user" at the beginning of the prerequisite (can't make a suggestion in the PR)

File: astro/install-aws.md [hunk 4]
```diff
@@ -100,7 +102,7 @@ From here, our team will provision an Astro cluster according to the specificati
 
 ## Step 3: Create a cross-account IAM role for Astro
 
-After your Astro Cluster is created, Astronomer provides you with an external ID. Save the external ID as a secret or in another secure format. You need to add the external ID to AWS to allow Astronomer to access your AWS resources. For more information, see [How to use an external ID when granting access to your AWS resources to a third party](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
+To create a cross-account IAM role for Astro you need an external ID that links to the resources Astronomer creates in your dedicated AWS account. To obtain your external ID, provide your [activation details](data-plane-activation.md#pre-flight-checklist) to [Astronomer support](https://support.astronomer.io). When you receive the external ID, save it as a secret or in another secure format. See [How to use an external ID when granting access to your AWS resources to a third party](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
```
Comment (jwitz) [inline astro/install-aws.md:105]:
```suggestion
To create a cross-account IAM role for Astro you need an external ID that links to the resources Astronomer creates in your dedicated AWS account. To obtain your external ID, provide your [activation details](data-plane-activation.md#pre-flight-checklist) to [Astronomer support](https://support.astronomer.io). When you receive the external ID from Astronomer, save it as a secret or in another secure format. See [How to use an external ID when granting access to your AWS resources to a third party](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
```

**Review by jwitz — APPROVED** [review]
LGTM!

**Pull request merged**